posthog/tasks/usage_report.py

```diff
@@ -344,7 +344,7 @@ def get_teams_with_billable_event_count_in_period(
         f"""
         SELECT team_id, count({distinct_expression}) as count
         FROM events
-        WHERE timestamp between %(begin)s AND %(end)s AND event != '$feature_flag_called' AND event NOT IN ('survey sent', 'survey shown', 'survey dismissed')
+        WHERE timestamp between %(begin)s AND %(end)s AND event != '$feature_flag_called' AND event NOT IN ('survey sent', 'survey shown', 'survey dismissed') AND (event != '$exception' OR mat_$exception_issue_id IS NULL)
```

Comment:
Just to make sure this is doing the right thing - 

If the event has
`event.event  == $exception` -- EXclude it
but if it has
`event. exception_issue_id == NULL` -- INclude it

so if I have an event with:
```
event.event  == $exception && event. exception_issue_id == NULL
```
then it would be included?

I feel like the way it's written is hard to parse. Can you see if there's a way to make it easier to read and if not add comments to explain? And also a test to make sure it's excluding things properly 🙏